unittests/good_mock_server_v1.2.0.py

```diff
@@ -19,6 +19,15 @@
 auth_basic = HTTPBasicAuth()
 auth_bearer = HTTPTokenAuth(scheme="Bearer")
 
+@auth_bearer.error_handler
```

Comment:
TODO: yash: check where this coming from?

Reply:
@alextsaihi 
After investigating, I have identified the source of this change. The commit was originally added to Emre's old unittests branch, and your branch picked it up since it is based on Emre's branch. In conclusion, these code changes are appropriately located and align with the intended structure.